### PR TITLE
button.jsx implemented

### LINKED ISSUE #4: Implement Button.jsx
image

Number of props: 2 text, link
Location: Any
Number of Images: 0
React Icon: No
Description: Component includes text in the center, border, and the link that opens up on click with a new tab.

Path: \src\components\Button.jsx

References: 
https://acm-ucr.github.io/docusaurus/docs/nextjs/next-link
https://acm-ucr.github.io/docusaurus/docs/react/props

### PR DESCRIPTION
**Pull request by jinnoh**
Closes #4

### CODE REVIEW

File: src/components/Button.jsx [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
+import Link from "next/link";
+
+const Button = ({ link, target }) => {
   return (
-    <div className="">
-      <p>Button</p>
-    </div>
+    <Link
+      className="h-20 w-13 text-4xl text-swim-black bg-swim-yellow px-10 py-5 font-bold text-xl text-center hover:opacity-50 active:opacity-40 z-10"
+      href="https://acm.cs.ucr.edu/"
```
Comment (seanquiambao) [inline src/components/Button.jsx:7]: put the link prop inside of href, do the same for target

File: src/components/Button.jsx [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
+      target="_blank"
+    >
+      <p className="align-middle"> Member Info Form</p>
```
Comment (seanquiambao) [inline src/components/Button.jsx:10]: replace "member info form" with a text prop.

File: src/components/Button.jsx [hunk 3]
```diff
@@ -1,8 +1,15 @@
-const Button = () => {
+import Link from "next/link";
+
+const Button = ({ link, target }) => {
   return (
-    <div className="">
-      <p>Button</p>
-    </div>
+    <Link
+      className="h-20 w-13 text-4xl text-swim-black bg-swim-yellow px-10 py-5 font-bold text-xl text-center hover:opacity-50 active:opacity-40 z-10"
```
Comment (seanquiambao) [inline src/components/Button.jsx:6]: width should be sized depending on text length.
https://tailwindcss.com/docs/width

File: src/components/Button.jsx (hunk 2 above)
Comment (seanquiambao) [reply src/components/Button.jsx:10]: remove the paragraph tags since you can put the text directly into the link tag.

**Review by seanquiambao — CHANGES_REQUESTED** [review]
make sure you link this pull request to your issue!

File: src/components/Button.jsx (hunk 1 above)
Comment (jinnoh) [reply src/components/Button.jsx]: I have taken care of the following comments, could you double check to see if all the issues have been resolved?

**Review by seanquiambao — APPROVED** [review]
looks good! i'll commit the suggestion since its a small fix

**Pull request merged**